src/current/_data/versions.csv

```diff
@@ -14,4 +14,4 @@ v22.2,2022-12-05,2023-12-05,2024-06-05,N/A,N/A,N/A,N/A,N/A,v22.1,release-22.2
 v23.1,2023-05-15,2024-05-15,2024-11-15,23.1.11,23.1.12,2023-11-13,2024-11-13,2025-11-13,v22.2,release-23.1
 v23.2,2024-02-05,2025-02-05,2025-08-05,23.2.6,23.2.7,2024-07-08,2025-07-08,2026-07-08,v23.1,release-23.2
 v24.1,2024-05-20,2025-05-20,2025-11-20,N/A,N/A,N/A,N/A,N/A,v23.2,release-24.1
-v24.2,N/A,N/A,N/A,N/A,N/A,N/A,N/A,N/A,v24.1,master
+v24.2,N/A,N/A,N/A,N/A,N/A,N/A,N/A,N/A,v24.1,release-24.2

```

Comment:
```suggestion
v24.2,N/A,N/A,N/A,N/A,N/A,N/A,N/A,N/A,v24.1,master
```
This will get you past the build error until the SQL grammar is branched in https://github.com/cockroachdb/generated-diagrams/tree/master/grammar_svg.